src/com/lftechnology/task/dec31/DistanceManager.java

```diff
@@ -0,0 +1,207 @@
+package com.lftechnology.task.dec31;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+/**
+ * Q2. Create a class called Distance that includes two private data members
+ * feet (type int) and inches (type float). Use a method setData to set values
+ * to instance variables. Provide a method display that displays the feet and
+ * inches, a member function addDistance for adding two distances, and a member
+ * function compareDistance for comparing two distances. Implement your program
+ * in Java.
+ * 
+ * @author Sanjib Maharjan
+ * 
+ */
+public class DistanceManager {
+	public static void main(String[] args) {
+		Scanner userInput = new Scanner(System.in);
+		Distance distanceInput1 = new Distance();
+		Distance distanceInput2 = new Distance();
+		System.out.println("Input first value for distance");
+		setDistanceInput(distanceInput1, userInput);
+		System.out.println("Input second value for distance");
+		setDistanceInput(distanceInput2, userInput);
+		userInput.close();
+		displayAddDistances(new Distance[] { distanceInput1, distanceInput2 });
+		displayCompareDistances(distanceInput1, distanceInput2);
+	}
+
+	/**
+	 * 
+	 * This method adds the arrays of distance objects{@link Distance} and
+	 * displays the result.
+	 * 
+	 * @param distances
+	 *            an array of distance object{@link Distance}
+	 * @see Distance
+	 */
+	public static void displayAddDistances(Distance... distances) {
+		float tempInches = 0;
+		int numDistance = distances.length;
+		for (int i = 0; i < numDistance; i++) {
```

Comment:
When iteration and it's subsequent count  doesn't matter, You should use , `foreach` loop , 

`for(Distance singleDistance : distances)`